src/common/commonutils/FileUtils.c

```diff
@@ -5,13 +5,15 @@
 
 char* LoadStringFromFile(const char* fileName, bool stopAtEol, void* log)
 {
+    const int initialSize = 1024;
+    int currentSize = 0;
     FILE* file = NULL;
-    int fileSize = 0;
     int i = 0;
     int next = 0;
     char* string = NULL;
+    char* temp = NULL;
 
-    if ((NULL == fileName) || (-1 == access(fileName, F_OK)))
+    if (false == FileExists(fileName))

```

Comment:
nit: `if (!FileExists(fileName))`

Reply:
Returns bool and if you look at other instances in this file, same explicit comparison is used.